tendrl/commons/utils/ansible_module_runner.py

```diff
@@ -31,26 +31,42 @@ class AnsibleRunner(object):
 
     """
 
-    def __init__(self, module_path, **kwargs):
+    def __init__(
+        self,
+        module_path,
+        publisher_id=None,
+        node_id=None,
+        socket_path=None,
+        **kwargs
+    ):
         self.module_path = modules.__path__[0] + "/" + module_path
+        self.socket_path = socket_path
+        self.publisher_id = publisher_id
```

Comment:
They are handled in Message-Event combo as in https://github.com/Tendrl/commons/blob/develop/tendrl/commons/message.py#L55 and https://github.com/Tendrl/commons/blob/develop/tendrl/commons/event.py#L28